sdcm/send_email.py

```diff
@@ -144,12 +222,26 @@ class LongevityEmailReporter(BaseEmailReporter):
               'nemesis_name', 'nemesis_details', 'test_id',
               'username', 'nodes']
 
-    def build_report(self, email_data):
+    def cut_report_data(self, report_data, attachments_data, reason):
+        if ['test_status'] in attachments_data and len(attachments_data['test_status']) > 2 and len(
+                attachments_data['test_status'][1]) > 101:
+            #  Reduce number of records in test_status[1] to 100
+            attachments_data['test_status'][1] = attachments_data['test_status'][1][:100]
+            attachments_data['test_status'][1].append('List of events has been cut due to the email size limit')
+            return report_data, attachments_data
+        if attachments_data is not None:
+            return report_data, None
+        return None, None
+
+    def build_report(self, report_data, template_str=None):
```

Comment:
`build_report` signuture is different then the one in the `Email` class, is that on purpose ?